app/views/project_information/show/_information_list.erb

```diff
@@ -35,6 +35,10 @@
       row.key { t('project_information.show.project_details.rows.advisory_board_date') }
       row.value { @project.advisory_board_date }
     end
+    summary_list.row do |row|
```

Comment:
The position of this really bugs me! :) could we move it to the bottom so the two advisory board items are together?